rollup-examples.config.js

```diff
@@ -30,7 +44,8 @@ function createOutput( file ) {
 			banner:
 				'/**\n' +
 				` * Generated from '${ path.relative( '.', inputPath.replace( /\\/, '/' ) ) }'\n` +
```

Comment:
The regex appears to be replacing `\` with `/`, but the output still shows `\` — https://github.com/gkjohnson/three.js/blob/rollup-jsm-examples-output/examples/js/loaders/GLTFLoader.js#L2. Is this working as intended?